src/scss/_index.scss

```diff
@@ -0,0 +1,22 @@
+hr{
+    border-top: 1px solid #000 !important;
+}
+
+.row{
```

Comment:
You should not change the margins for `row` as these are very important to how bootstrap functions. You can edit the `column gutters` in variables if you want to change how this looks.

Reply:
Removed these style, found out I didn't need them.